pentaho-karaf-assembly/src/main/filtered-resources/etc/custom.properties

```diff
@@ -274,7 +274,7 @@ org.osgi.framework.system.packages.extra= \
  org.pentaho.di.job.entry, \
  org.pentaho.di.www, \
  org.springframework.security;version=2.0.5.RELEASE, \
- org.springframework.security.context, \
+ org.springframework.security.context;version=2.0.5.RELEASE, \
```

Comment:
That's fine. I'm about to consolidate us back to one custom.properties. We'll get it then